tests/components/pushbullet/test_notify.py

```diff
@@ -1,301 +1,273 @@
-            notify.DOMAIN: {
-                "name": "test",
-                "platform": "pushbullet",
-                "api_key": "MYFAKEKEY",
-            }
-        }
-        with assert_setup_component(1) as handle_config:
-            assert setup_component(self.hass, notify.DOMAIN, config)
-        assert handle_config[notify.DOMAIN]
-        mock.register_uri(
-            requests_mock.POST,
-            "https://api.pushbullet.com/v2/pushes",
-            status_code=200,
-            json={"mock_response": "Ok"},
-        )
-        data = {
-            "title": "Test Title",
-            "message": "Test Message",
-            "target": ["device/DESKTOP", "email/user@host.net"],
-        }
-        self.hass.services.call(notify.DOMAIN, "test", data)
-        self.hass.block_till_done()
-        assert mock.called
-        assert mock.call_count == 2
-        assert len(mock.request_history) == 2
+    data = {
+        "title": "Test Title",
+        "message": "Test Message",
+        "target": ["device/DESKTOP"],
+    }
+    await hass.services.async_call(notify.DOMAIN, "test", data)
+    await hass.async_block_till_done()
+    assert requests_mock.called
+    assert requests_mock.call_count == 1
 
-        expected_body = {
-            "body": "Test Message",
-            "device_iden": "identity1",
-            "title": "Test Title",
-            "type": "note",
-        }
-        assert mock.request_history[0].json() == expected_body
-        expected_body = {
-            "body": "Test Message",
-            "email": "user@host.net",
-            "title": "Test Title",
-            "type": "note",
+    expected_body = {
+        "body": "Test Message",
+        "device_iden": "identity1",
+        "title": "Test Title",
+        "type": "note",
+    }
+    assert requests_mock.last_request.json() == expected_body
+
+
+async def test_pushbullet_push_devices(hass, requests_mock, mock_pushbullet):
+    """Test pushbullet push to default target."""
+    config = {
+        notify.DOMAIN: {
+            "name": "test",
+            "platform": "pushbullet",
+            "api_key": "MYFAKEKEY",
         }
-        assert mock.request_history[1].json() == expected_body
+    }
+    with assert_setup_component(1) as handle_config:
+        assert await async_setup_component(hass, notify.DOMAIN, config)
+        await hass.async_block_till_done()
+    assert handle_config[notify.DOMAIN]
+    requests_mock.register_uri(
+        "POST",
+        "https://api.pushbullet.com/v2/pushes",
+        status_code=200,
+        json={"mock_response": "Ok"},
+    )
+    data = {
+        "title": "Test Title",
+        "message": "Test Message",
+        "target": ["device/DESKTOP", "device/My iPhone"],
+    }
+    await hass.services.async_call(notify.DOMAIN, "test", data)
+    await hass.async_block_till_done()
+    assert requests_mock.called
+    assert requests_mock.call_count == 2
+    assert len(requests_mock.request_history) == 2
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6285789)